css/index.css

```diff
@@ -0,0 +1,66 @@
+
+body {
+    background-color: rgb(142, 142, 158);
+    color: white;
+    justify-content: center;
+    align-items: center;
```

Comment:
It doesn't look like the body elements are being center aligned. This is because `display: flex` is commented out below. Something to check out before submitting next week's assignment.